### ISSUE TITLE
Wiki error client.md

### ISSUE THREAD

**Issue by Sanjay-B** [SinisterRectus/Discordia#45]
```
+| setStatusIdle()              | WS          | Sets the user status to idle. Warning: This can silently fail!                                 |
+| setStatusOnline()            | WS          | Sets the user status to idle. Warning: This can silently fail!
```

Should be: 
```
+| setStatusIdle()              | WS          | Sets the user status to idle. Warning: This can silently fail!                                 |
+| setStatusOnline()            | WS          | Sets the user status to online. Warning: This can silently fail!                                 |
```

**Comment by SinisterRectus** [comment]
The warning is correct for both.

**Comment by SinisterRectus** [comment]
I thought you had only removed the warning. That should not be removed. We will fix the rest for the next release.

**Comment by Sanjay-B** [comment]
Sounds good. Just thought you should know.

**Comment by Tigerism** [comment]
Typo fixed as of https://github.com/SinisterRectus/Discordia/pull/47

**Comment by SinisterRectus** [comment]
Fixed by cfa0f1a7d7266fbe90c600db92cf54287ce5925a
